tock/tock/templates/employees/user_form.html

```diff
@@ -17,7 +17,18 @@ <h3 class="chart__title">Project Hours Over Time</h3>
 </figure>
 
 <form action="" method="post">{% csrf_token %}
-    {{ form.as_p }}
+    <!--{{ form.as_p }}-->
+    {{ form.first_name.label_tag }}
+    {{ form.first_name }}
+    {{ form.last_name.label_tag }}
+    {{ form.last_name }}
+    {{ form.start_date.label_tag }}
+    {{ form.start_date }}
+    {{ form.end_date.label_tag }}
+    {{ form.end_date }}
+    {{ form.current_employee }}
+    {{ form.current_employee.label_tag }}
+
```

Comment:
Making the form more flexible by not using `{{ form.as_p }}` so the label for the checkbox can be after the input (per the USWDS)